client/galaxy/scripts/mvc/workflow/workflow.js

```diff
@@ -0,0 +1,201 @@
+/** Workflow view */
+define( [ 'mvc/form/form-view',
+          'mvc/ui/ui-misc',
+          'mvc/ui/ui-select',
+          'mvc/tool/tool-form-composite' ], function( Form, Ui, Select, ToolForm ) {
```

Comment:
Sure!  Pending that, I'm happy to merge this.  Don't worry about rebuilding the client again if there are no additional changes, I can resolve that in the merge.